specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/MultivariateAnomalyDetector.json

```diff
@@ -0,0 +1,1124 @@
+            0.0860302299261093,
+            0.08287354558706284,
+            0.0823553800582885
+          ],
+          "items": {
+            "type": "number"
+          }
+        },
+        "validationLosses": {
+          "type": "array",
+          "example": [
+            0.9573427438735962,
+            0.3182229995727539,
+            0.3402646481990814,
+            0.2765348851680755,
+            0.2574014365673065,
+            0.269286036491394,
+            0.2637230455875397,
+            0.2590833008289337,
+            0.2597166895866394,
+            0.2597350180149078
+          ],
+          "items": {
+            "type": "number"
+          }
+        },
+        "latenciesInSeconds": {
+          "type": "array",
+          "example": [
+            0.3398594856262207,
+            0.3659665584564209,
+            0.3736064434051513,
+            0.3513407707214355,
+            0.3370304107666015,
+            0.3187627792358398,
+            0.3283309936523437,
+            0.3503587245941162,
+            0.3080024719238281,
+            0.3327946662902832
+          ],
+          "items": {
+            "type": "number"
+          }
+        }
+      },
+      "type": "object"
+    },
+    "VariableState": {
+      "properties": {
+        "variable": {
+          "type": "string",
+          "description": "Variable name."
+        },
+        "filledNARatio": {
+          "type": "number",
+          "description": "Proportion of NaN values filled of the variable.",
+          "minimum": 0.0,
+          "maximum": 1.0
+        },
+        "effectiveCount": {
+          "type": "integer",
+          "description": "Number of effective points counted."
+        },
+        "firstTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "First valid timestamp of the variable."
+        },
+        "lastTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last valid timestamp of the variable."
+        }
+      },
+      "type": "object"
+    },
+    "Model": {
+      "required": [
+        "createdTime",
+        "lastUpdatedTime",
+        "modelId"
+      ],
+      "properties": {
+        "modelId": {
+          "type": "string",
+          "format": "uuid",
+          "description": "Model identifier.",
+          "example": "45aad126-aafd-11ea-b8fb-d89ef3400c5f"
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time (UTC) when the model was created.",
+          "example": "2020-06-30T00:00:00Z"
+        },
+        "lastUpdatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Date and time (UTC) when the model was last updated.",
+          "example": "2020-06-30T00:00:00Z"
+        },
+        "modelInfo": {
+          "$ref": "#/definitions/ModelInfo"
+        }
```

Comment:
I wish we had caught this earlier -- there's really no need to nest properties inside a "modelInfo" field. This whole schema should be "information about a model". But might not be something we can fix at this point.

Reply:
Won't fix in this version.